Makefile

```diff
@@ -61,6 +61,7 @@ docker_build_master:
 
 .PHONY: test
 test: test-unit
+	make test-integration
```

Comment:
I would rather `make test` only runs the unit tests. This will keep our project consistent with the existing Mobile Services. Consistency is key here.

Reply:
@davidffrench,

So, what name you suggest to run all tests? Could it be all-tests?

Reply:
Ya, that is perfectly fine.

Reply:
Change done as requested.